src/file_handler.rs

```diff
@@ -264,7 +265,7 @@ where
     T: DeserializeOwned,
 {
     /// Read the contents of the file and decode it as JSON.
-    #[cfg_attr(feature = "cargo-clippy", allow(redundant_closure))] // because of lifetimes
+    #[allow(clippy::redundant_closure)] // because of lifetimes
```

Comment:
I wonder if we can fix this clippy error now due to NLL, introduced in 1.31. It would obviate the need for this explicit `allow`.